Doc/library/unittest.rst

```diff
@@ -2358,9 +2361,10 @@ These must be implemented as class methods::
         def tearDownClass(cls):
             cls._connection.destroy()
 
-If you want the ``setUpClass`` and ``tearDownClass`` on base classes called
-then you must call up to them yourself. The implementations in
-:class:`TestCase` are empty.
+The implementations in :class:`TestCase` are present, but empty.
+That means the implemenation is executed for classes descending from
```

Comment:
Choosing "descending" instead of "derived" or "child", to suggest the inheritance graphs can be complicated.